Formula/rome.rb

```diff
@@ -0,0 +1,31 @@
+class Rome < Formula
+  desc "Carthage cache for S3, Minio, Ceph, Google Storage, Artifactory and many others"
+  homepage "https://github.com/tmspzz/Rome/#readme"
+  url "https://github.com/tmspzz/Rome/archive/refs/tags/v0.24.0.65.tar.gz"
+  sha256 "7aee4de208a78208559d6a9ad17788d70f62cace4ff2435b3e817a3e03efdef6"
+  license "MIT"
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc@8.10" => :build
```

Comment:
```suggestion
  depends_on "ghc@8.10" => :build
  uses_from_macos "zlib"
```
This should help with the Linux failure.

Reply:
Thanks! Fixed!